docs/_pages/httpresponsemessages.md

```diff
@@ -0,0 +1,28 @@
+---
+title: HttpResponseMessages
+permalink: /httpresponsemessages/
+layout: single
+classes: wide
+sidebar:
+nav: "sidebar"
```

Comment:
If this should be consistent with other markdown files.
```suggestion
  nav: "sidebar"
```

Reply:
Was related to the other mistake